<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Application shortcuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Speak highlighted text</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <widget class="QCheckBox" name="speakCtrlCheckBox">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="speakAltCheckBox">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="speakClearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item>
           <widget class="QCheckBox" name="stopCtrlCheckBox">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="stopAltCheckBox">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopClearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Activate window</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
          <item>
           <widget class="QCheckBox" name="activateCtrlCheckBox">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="activateAltCheckBox">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="activateClearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clipboardCheckBox">
     <property name="text">
      <string>Speak clipboard content</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Close on system tray</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Start minimized</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
